crm_task/models/project_task.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2020 Adgensee - Vincent Garcies
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class LeadinTask(models.Model):
+    _inherit = 'project.task'
+
+    lead_id = fields.Many2one("crm.lead", string="Opportunity", required=False)

```

Comment:
why the required false? by default is false.

Reply:
I just deleted that, thanks for your feedback